src/components/Header/Header.js

```diff
@@ -0,0 +1,605 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { FiGlobe } from 'react-icons/fi';
+import { IoPersonCircle } from 'react-icons/io5';
+import { BiSearch } from 'react-icons/bi';
+import Calendar from '../Calendar';
+import ProfileModal from './ProfileModal';
+import LoginModal from './LoginModal';
+import SearchModal from './SearchModal';
+import LoginButton from '../Login/Button';
+
+const Header = ({ isMain }) => {
```

Comment:
```suggestion
const Header = ({ isMainPage }) => {
```

제가 정했던거지만.. 다시보니 뒤에 Page 붙혀주면 조금 더 명시적일거같아요!